ingestion/src/metadata/antlr/split_listener.py

```diff
@@ -21,10 +21,10 @@ class FqnSplitListener(FqnListener):
     def __init__(self):
         self._list = []
 
-    def enterQuotedName(self, ctx: FqnParser.QuotedNameContext):
+    def enter_quoted_name(self, ctx: FqnParser.QuotedNameContext):
```

Comment:
we'll need to double-check why the server is unable to start. I am not sure if we might be overlooking anything by updating this method names since those are related to ANTLR.

I'd suggest to revert this changes and see if the py-tests pass properly, since the major topic in the github issue was about cleaning up the Makefile (which is beautifully done here already).

Thanks